welcome/overview/is-it-for-you.md

```diff
@@ -1,37 +1,34 @@
-[title]: - "Is High Throughput Computing for You?"
+[title]: - "Is the Open Science Grid for You?"
 
 
-## Ideal Charateristics for Distributed High Throughput Computing (HTC)
+The [Open Science Grid][osg] is a national network that offers a strategic advantage 
+for computing work that can be run as numerous short tasks. 
 
--   Your science workflow can be split into independent jobs.
--   These jobs can be run on a single processor or compute node.
--   Your application software is “portable”. (We can help make applications portable and also provide pre-installed software modules)
 
+|   		| **Ideal jobs!**| **Still very advantageous** | **Less-so, but maybe** | 
+|:----------|:--------------|:--------------|:--------------|
+| CPU		|	1			|	<8			|	>8 (or MPI)	|
+| Walltime	| 	<12 hrs		|	<24 hrs		|	>24 hrs		|
+| RAM		| 	< few GB	|	<10s GB		|	>10s GB		|
+| Input		| 	<500 MB		|	<10 GB		|	>10 GB		|
+| Output	| 	< 1GB		|	<10 GB		|	>10 GB		|
+| Software	| pre-compiled binaries, containers | Most other than -> | Licensed software, non-Linux |
 
+For problems that can be run as many independent jobs, as in the first two columns 
+of the table above, the OSG provides practically-unlimited resources, with the possibility 
```

Comment:
"... practically-unlimited resources as compared to local computing systems, with ..."